Dockerfile

```diff
@@ -2,20 +2,34 @@
 
 FROM golang:1.21-alpine3.18 as build
 
+ARG ARCH=x86_64
+ARG WASM=false
```

Comment:
this is the switch we need to pull when we are ready to roll it out? (+ corresponding values in other docker files)

Reply:
Yep. The gh build actions have an input to override this for now